### ISSUE TITLE
[Feature Request] Use OR logical operator for alarm management in device profiles

### ISSUE THREAD

**Issue by elietaillardat** [thingsboard/thingsboard#7227]
We would like to have the ability to add an alarm condition with the logical operator OR (today it seems only possible to do it with the AND operator) for checks on different attributes/telemetries.

Example : buttonAlertSeverity == 'critical' or buttonAlertNoPush == true (with buttonAlertSeverity and buttonAlertNoPush being 2 different telemetries)

Thanks

**Comment by Backdraft007** [comment]
In the alarm rules you can add more create conditions. One condition for buttonAlertSeverity and a second condition for buttonAlertNoPush.
It is like an OR operator.

**Comment by elietaillardat** [comment]
Yes but if I create a new condition, **I can't assign it to the same alarm level**.
In my case I need two conditions for a critical alarm, but I'm forced to use the major state, which is not suitable.

**Comment by Backdraft007** [comment]
Yes, sorry. I didn't see that.

**Comment by ashvayka** [comment]
We plan to significantly improve alarm rules in 3.5 or 3.6 at the latest. Will add this support as well.

**Comment by mykhailokornieiev** [comment]
Hi. Thank you for the suggestion. Such a feature request was previously added to our backlog for developers. So far, there is no estimated time for this improvement. Please follow our Release Notes to see all new additions and fixes to the platform in subsequent releases.
We're closing this ticket to keep our repository well-maintained and focused on active issues.
We appreciate your patience and understanding.

**Comment by mykhailokornieiev** [comment]
Hi. 
I'm glad to inform you that this feature was implemented as a part of Alarm Rules 2.0 in v4.3 release.
Feel free to open a new ticket in case any questions or issues arise.
Have a great day
